docs/host/host.md

```diff
@@ -42,6 +52,8 @@ Before removing a node from a Harvester cluster, determine if the remaining node
 
 If the remaining nodes do not have enough resources, VMs might fail to migrate and volumes might degrade when you remove a node.
 
+If you have some volumes which were created from the customized `StorageClass` with the value **1** of the [Number of Replicas](../advanced/storageclass.md#number-of-replicas), it is recommended to backup those single-replica volumes or re-deploy the related workloads to other node in advance to get the volume scheduled to other node. Otherwise, those volumes can't be rebuilt or restored from other nodes after this node is removed.
```

Comment:
This is a good argument for retaining the existing behavior of `block-if-contains-last-replica` instead of implementing the change requested by https://github.com/longhorn/longhorn/issues/9188. If we allow the drain even though the drained node contains an "unprotected" replica, and then the customer feels it is safe to remove the node, their data is gone.